website/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
```

Comment:
Not sure if you wanna update this to not include the above

Reply:
i just left that readme as is since the system is using the README.md in cips directory